src/components/ProfileMenu/index.tsx

```diff
@@ -70,7 +70,7 @@ export default function ProfileMenu({ lang }: ProfileMenuProps) {
           <IconButton
             data-testid="avatarIconButton"
             size="small"
-            sx={{ ml: 2, paddingTop: '15px' }}
+            sx={{ ml: 2, padding: '0.5rem', borderRadius: 0 }}
```

Comment:
```suggestion
            sx={{ ml: 2, padding: '0.5rem' }}
```

Perhaps rather keep the border. The oval shadow shape is by design, it was intended to balance out the overall geometry. When the design was implemented, some parts of the app didn't receive the planned layout from the prototype (the user stories were related mainly to the main view / search controls - which are implemented to certain degree, but for example user profile view hasn't been updated as there hasn't been user story to fix it with).